oci/private/tarball.bzl

```diff
@@ -0,0 +1,98 @@
+"""creates tarball from oci_image that can be loaded by runtimes such as podman and docker"""
+
+load(
+    "@rules_pkg//pkg/private:pkg_files.bzl",
+    "add_single_file",
+    "add_tree_artifact",
+    "write_manifest",
+)
+
+doc = """Creates tarball from OCI layouts that can be loaded into docker daemon without needing to publish the image first.
+
+Passing anything other than oci_image to the image attribute will lead to build time errors.
+
+example;
+
+```shell
+bazel build //target
+docker load --input $(bazel cquery --output=files //target)
+```
+"""
+
+attrs = {
+    "image": attr.label(mandatory = True, allow_single_file = True, doc = "Label to an oci_image target"),
```

Comment:
do you have the pre-commit installed? I would have expected this to be formatted differently

Reply:
I have it but still didn't format the line to be multi.